### DISCUSSION TITLE
Welcome to deployments-k8s Discussions!

### DISCUSSION THREAD

**Discussion by denis-tingaikin** [networkservicemesh/deployments-k8s#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by denis-tingaikin** [comment]
@edwarnicke 
We have implemented an alternative realization of intregration tests. 
The main idea is use `markdown files` as a source for tests:
In short how it is works:

1.  We are using a markdown files as a source of the tests
Example: https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/basic
That I hope is user-readable.
Also those examples based on kustomize k8s feature.
2.  We just parse those examples and generate a golang suites:https://github.com/networkservicemesh/integration-tests/blob/main/suites/basic/suite.gen.go via  https://github.com/networkservicemesh/gotestmd

Finally we have user-reproduciable tests that  can be covered on ci.

**Reply by edwarnicke** [reply]
Woot!

**Comment by denis-tingaikin** [comment]
@edwarnicke 
I think that it would be nice to put somewhere our docs about repo pipeline for folks. 
What do you think?

**Reply by edwarnicke** [reply]
Please do... did you have a place in mind?
